Specs/Scene/TextureAtlasSpec.js

```diff
@@ -274,10 +283,10 @@ defineSuite([
         });
 
         var promises = [];
-        promises.push(atlas.addImage(greenImage.src, greenImage));
-        promises.push(atlas.addImage(blueImage.src, blueImage));
-        promises.push(atlas.addImage(bigRedImage.src, bigRedImage));
-        promises.push(atlas.addImage(bigBlueImage.src, bigBlueImage));
+        promises.push(atlas.addImage(guidArray[0], greenImage));
```

Comment:
What were the changes in this file fore, this is much more obtuse than the previous code.  Assuming the change was necessary, having 4 variables (greenGuid, etc..) would make the tests much more readable.  These indexes amount to magic numbers otherwise.

Reply:
This is necessary because `ImageBitmap` has no `src` property. This way it works regardless of whether the image is `Image` or `ImageBitmap`. 

You're right that just having `greenGuid` etc. variables looks much more readable.